### ISSUE TITLE
Any pre-trained weights provided?

### ISSUE THREAD

**Issue by DawnChou** [alexandrosstergiou/keras-DepthwiseConv3D#11]
Hey thank you for your work. Have you evaluated this model on any video datasets like UCF-101 or HMDB-51, and how much the accuracy is? If you evaluate it on bigger dataset like Kinetics, could you please provided a pre-trained weights, I really need it because of the limitation of computing resource. Thank you!

**Comment by alexandrosstergiou** [comment]
Hi @DawnChou. The repo is meant for a quick easy-to-use solution for people using Keras explicitly. I have not used the code for actual training nor I a plan to in the future, simply because other libraries (e.g. PyTorch) provide both greater flexibility in terms of implemented/customisable layers and in general have greater speedups. I can however definitely recommend using PyTorch w/ Nvidia's Apex in half precision if there are constraints in terms of resources.

Best,
Alex
